applications/vanilla/views/modules/categories.php

```diff
@@ -44,6 +44,12 @@
                     $CountText = '';
                 }
 
+                if( $Category->PhotoUrl !== "" ){
+                    echo '<div class="Item-Icon PhotoWrap PhotoWrap-Category">';
+                    echo '<img src="' .$Category->PhotoUrl. '" class="CategoryPhoto" alt="' .$Category->Name. '">';
```

Comment:
This would be an XSS. Escape the name or use twig which will do it automatically.

Reply:
It's a big scoop to convert the file to twig